adapters/engagebdr/engagebdrtest/supplemental/multi-banner-video.json

```diff
@@ -0,0 +1,108 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
```

Comment:
Both this file and the previous one `multi-1-invalid-imp.json` seem identical. Probably a leftover by mistake. Can you please remove one of them or if you mean to add a different test case in one of these files then can you please update that?

Reply:
@yannaingkyaw1986 pinging to follow up on the above comment

Reply:
updated